sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceClient.java

```diff
@@ -344,7 +347,8 @@ public Response<Void> addConnectionToGroupWithResponse(String group, String conn
      *
      * @param hub Target hub name, which should start with alphabetic characters and only contain alpha-numeric
      * characters or underscore.
-     * @param groupsToAdd Target groups and connection filter.
+     * @param groups Target group names
```

Comment:
I see you send `null` in test. Please clarify this in Javadoc (i.e., what would be the effect of the API, if this param is null).

Same to the Async client.